### ISSUE TITLE
 outdoor-rain test set

### ISSUE THREAD

**Issue by CathyM123** [fingerk28/Two-stage-Knowledge-For-Multiple-Adverse-Weather-Removal#18]
Do you have the outdoor-rain test set? I downloaded it from the original paper and only saw the training set, not the test set. If not, how did you deal with it?
